spec/array_database_spec.rb

```diff
@@ -40,6 +40,34 @@
     end
   end
 
+  describe '#search' do
+    it 'returns any contacts that matches string given' do
+      second_contact = {
+        name: 'Oscar Wilde',
+        address: 'Paris',
+        phone: '00000000000',
+        email: 'oscar@wilde.com',
+        notes: 'I think he has an oscar'
```

Comment:
if you want to make sure that it works for downcase too, I would also downcase the o in `Oscar Wilde`, in case the name is checked first when you loop through the hash (while arrays are looped in a specific order, hashes aren't ordered at all).

Then you have the `test_details` as the example with an **uppercase** letter and the `second_contact` as the example with the **downcase** letter